### PR TITLE
Add tabindex option

### PR DESCRIPTION
**Pull request by MiketoString**

### CODE REVIEW

File: src/trumbowyg.js [hunk 1]
```diff
@@ -338,7 +338,8 @@ jQuery.trumbowyg = {
                 .addClass(prefix + 'editor')
                 .attr({
                     'contenteditable': true,
-                    'dir': t.lang._dir || t.o.dir
+                    'dir': t.lang._dir || t.o.dir,
+                    'tabindex': t.tabindex || -1
```
Comment (Alex-D) [inline src/trumbowyg.js]: I'm not sure about that. By default, we don't want this attribute. Add it only if exists.

I maybe fix that ASAP or if you want, you can do it :)

**Comment by MiketoString** [comment]
Hi Alex, just pushed a change to only add the tabindex attribute if the option is specified.

**Comment by Alex-D** [comment]
Yeah! Can you squash your commits to keep a clean log?

Thanks :)

**Comment by MiketoString** [comment]
squashed

**Comment by Alex-D** [comment]
Thanks! :)

**Pull request merged**